src/poetry/core/_vendor/jsonschema/LICENSE

```diff
@@ -1,19 +0,0 @@
-Copyright (c) 2012 Julian Berman
```

Comment:
`vendoring` removed this, but note there is already a [`COPYING`](https://github.com/python-poetry/poetry-core/blob/master/src/poetry/core/_vendor/jsonschema/COPYING)